robot/packages/utils.go

```diff
@@ -252,7 +252,8 @@ func commonCleanup(logger logging.Logger, expectedPackageDirectories map[string]
 			continue
 		}
 
-		// There should be no non-dir files in the packages/data dir except .status.json files. Delete any that exist
+		// There should be no non-dir files in the packages/data dir
+		// except .status.json and .first_run_succeeded. Delete any that exist
```

Comment:
I'm not sure if .status.json or .first_run_succeeded files should be at this level? .first_run_succeeded definitely not, since they are in .viam/packages/data/module, though unclear if we download other things into this top level with the corresponding .status.json file

Reply:
based on how getSyncFileName works, I don't think .status.json files will ever be at the packages/data dir level (LocalDataDirectory joins LocalDataParentDirectory with the package name, and LocalDataParentDirectory is already `.viam/packages/data/XXX`)